components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -179,6 +180,14 @@ abstract class EngineSession(
      */
     abstract fun clearFindMatches()
 
+    /**
+     * Enables/disables fullscreen mode. It also invokes a callback when fullscreen mode is exited.
+     * For GeckoView, this shouldn't necessarily be used to
+     *
+     * @param enable Turns fullscreen mode on/off.
+     */
+    abstract fun toggleFullScreenMode(enable: Boolean)
+
```

Comment:
`toggleFullScreenMode` doesn't seem to be the right thing here: It never goes to fullscreen mode. In GeckoView it only exists full screen mode. In WebView it does nothing. Should this only be a method that exits full screen mode? And shouldn't we invoke the WebView callback in the `browser-engine-system` implementation`

Reply:
You're right. Since we're hiding the WebView implementation in the screen this method doesn't follow the original idea. Sorry, I should have thought about this earlier.